code/integration-tests/runtime-tests/tests/utils/ibcUtils.ts

```diff
@@ -0,0 +1,306 @@
+  const txs = [
+    api.tx.sudo.sudo(api.tx.ibc.addChannelsToFeelessChannelList(0, 1)),
+    api.tx.sudo.sudo(api.tx.ibc.addChannelsToFeelessChannelList(1, 0)),
+    api.tx.sudo.sudo(api.tx.ics20Fee.addChannelsToFeelessChannelList(0, 1)),
+    api.tx.sudo.sudo(api.tx.ics20Fee.addChannelsToFeelessChannelList(1, 0)),
+  ]
+  await sendWithBatchAndWaitForSuccess(
+    api,
+    sudoKey,
+    api.events.utility.BatchCompleted.is,
+    txs,
+    false
+  );
+}
+
+export function createIbcTransferExtrinsic(
+  api: ApiPromise, receiver: string,
+  asset: number | string,
+  amount: string,
+  channelId: number,
+  toChannel: string,
+  memo: string,
+): SubmittableExtrinsic<"promise"> {
+  let params;
+  if (toChannel === 'centauri' || toChannel === 'osmosis') {
+    params = api.createType('PalletIbcTransferParams', {
+      to: api.createType('PalletIbcMultiAddress', {
+        Raw: api.createType('Bytes', receiver),
+      }),
+      sourceChannel: api.createType('u64', channelId),
+      timeout: api.createType('IbcPrimitivesTimeout', {
+        Offset: {
+          timestamp: '30000',
+          height: api.createType('Option<u64>', 50),
+        }
+      })
+    });
+  } else {
+    params = api.createType('PalletIbcTransferParams', {
+      to: api.createType('PalletIbcMultiAddress', {
+        id: api.createType('AccountId32', receiver)
+      }),
+      sourceChannel: api.createType('u64', channelId),
+      timeout: api.createType('IbcPrimitivesTimeout', {
+        Offset: {
+          timestamp: '30000',
+          height: api.createType('Option<u64>', 50),
+        }
+      })
+    });
+  }
+  const assetId = api.createType('u128', asset);
+  const amountTobeSent = api.createType('u128', amount);
+  return api.tx.ibc.transfer(params, assetId, amountTobeSent, memo);
+}
+
+export async function queryAllBalanceOnCentauri(address: string) {
+  const {stdout} = await exec(`~/go/bin/centaurid query bank balances ${address} --output json`);
+  const formattedBalance = JSON.parse(stdout);
+  const balances = parseBalancesFromCentauri(formattedBalance.balances);
+  return balances;
+}
+
+export async function initiateIbcFromCentauri(sender: string, receiverAddress: string) {
+  await
+    exec(`~/go/bin/centaurid tx ibc-transfer transfer transfer channel-0 "${receiverAddress}" 100000000000ibc/7880ADBE343B13930823D0A186951351AA0A3FA900F13AEA6A3E0746B02FE521 --from ${sender} --gas-adjustment=1.5 -y;`);
+  await waitForSeconds(8);
+  await
+    exec(`~/go/bin/centaurid tx ibc-transfer transfer transfer channel-0 "${receiverAddress}" 10000000000000ibc/4859E46FF89C3A8D361A2F69CB04A2F5CD9D2CB01171E9D91B9E36405B89318A --from ${sender} --gas-adjustment=1.5 -y;`);
+  await waitForSeconds(8);
+  await
+    exec(`~/go/bin/centaurid tx ibc-transfer transfer transfer channel-0 "${receiverAddress}" 10000000ibc/449BE1BB6F508F0CBE44BCBD810C7A9A1DCEC41B1F4880802A7A79FF4FA8DE53 --from ${sender} --gas-adjustment=1.5 -y;`);
```

Comment:
just do `go` NOT `~/go/bin` I will not run on my machine no on CI.

Reply:
cosmjs has good cosmos SDK btw. may be use.  but I am 100% with cli. But use `go`, not `~/go/bin/centauridi` .

binary must be in path when test starts to be easy run everywhere

Reply:
changed to centaurid and osmosisd. I think as long as they are set executable on ci, it should be good to run.